storage/txcache/txListForSenderAsSortedMapItem.go

```diff
@@ -97,3 +103,8 @@ func (listForSender *txListForSender) GetScoreChunk() *maps.MapChunk {
 func (listForSender *txListForSender) SetScoreChunk(scoreChunk *maps.MapChunk) {
 	listForSender.scoreChunk = scoreChunk
 }
+
+// GetScoreChangeInProgressFlag gets the atomic flag indicating whether a score change is in progress
+func (listForSender *txListForSender) GetScoreChangeInProgressFlag() *atomic.Flag {
```

Comment:
Suggestion to drop the Get part (not go idiomatic)

Reply:
Fixed.